OpenTrans.net/Feature.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace OpenTrans.net
+{
+    public class Feature
+    {
+        public string FName { get; set; }
```

Comment:
does it make sense to call it FeatureName/ FeatureValue etc.?

Reply:
personally, i don't like any prefixes if the context is implicitly clear, i.e.  feature.Name.
But in case of mapping context its fine to use same names, using convenient naming convention, i.e. camel case here.
So, how do you want it to be?

Reply:
Sounds reasonable, thanks! So just 'Name', 'Value' etc. ?

Reply:
I can remove the F-prefix here, but consider the possibility that future versions of the format may introduce fields which will collide with existing, e.g. VALUE with different semantics than FVALUE, that's why i'm for same names but different casing?
Ok, in this example, it's very unlikely of course, but it's about some general naming here, right? ;) 
So it's up to you...

Reply:
Let's skip 'F' and just go for Name etc.
Would you implement it or shall I?